1-js/02-first-steps/15-function-expressions-arrows/1-rewrite-arrow/solution.md

```diff
@@ -1,9 +1,8 @@
 
 ```js run
-function ask(question, yes, no) {
-  if (confirm(question)) yes()
-  else no();
-}
+*!*
```

Comment:
Well, the phrasing for the exercise was `Replace Function Expressions with arrow functions in the code`

So my solution was about using as many arrow functions as possible, including the first function that is *not* a function expression; so now that I read this again I might be wrong, and the initial solution is maybe better